### PR TITLE
Improve documentation of SwitchStatement

### PR DESCRIPTION
**Pull request by wilzbach**
Follow-up to https://github.com/dlang/dmd/pull/7842

### CODE REVIEW

File: src/dmd/statement.d [hunk 1]
```diff
@@ -1591,6 +1591,15 @@ extern (C++) final class SwitchStatement : Statement
...
     {
+        /*
+        * Checks the scope of a label for existing variable declaration.
+        *
+        * Params:
+        *   vd = variable declaration to check
+        *
+        * Returns: `true` if the variables declared in this label would be skipped
+                   `false` otherwise
+        */
```
Comment (ntrel) [inline src/dmd/statement.d]: Seems a bit verbose, especially for a non-ddoc symbol. Does  `Params` really add anything?

**Review by ntrel — APPROVED** [review]

File: src/dmd/statement.d (hunk 1 above)
Comment (jacob-carlborg) [reply src/dmd/statement.d]: I think it _should_ have a Ddoc comment.

Comment (WalterBright) [reply src/dmd/statement.d]: The "`false` otherwise" is redundant and can be removed.

File: src/dmd/statement.d [hunk 2]
```diff
@@ -1591,6 +1591,15 @@ extern (C++) final class SwitchStatement : Statement
 
     final bool checkLabel()
     {
+        /*
+        * Checks the scope of a label for existing variable declaration.
+        *
```
Comment (WalterBright) [inline src/dmd/statement.d]: These blank lines unnecessarily take up vertical space.

**Comment by wilzbach** [comment]
Comments addressed - this adds more comments to an existing function, it's not rocket science :/
(FWIW I didn't even add the function ...)

File: src/dmd/statement.d [hunk 3]
```diff
@@ -1591,6 +1591,12 @@ extern (C++) final class SwitchStatement : Statement
 
     final bool checkLabel()
     {
+        /*
+        * Checks the scope of a label for existing variable declaration.
+        * Params:
+        *   vd = variable declaration to check
+        * Returns: `true` if the variables declared in this label would be skipped.
```
Comment (JinShil) [inline src/dmd/statement.d]: `grep`ping dmd source code revealed the prevailing style is to put the `Returns` description on the following line.

**Review by JinShil — APPROVED** [review]

File: src/dmd/statement.d (hunk 3 above)
Comment (wilzbach) [reply src/dmd/statement.d]: I like that too, but Walter's comment was:

> These blank lines unnecessarily take up vertical space.

Comment (JinShil) [reply src/dmd/statement.d]: 😆  My comment was on Walter's behalf.  Ok, then I'm good.

**Pull request merged**